### ISSUE TITLE
systemFont with weight doesn't work

### ISSUE THREAD

**Issue by ziryanov** [evermeer/AttributedTextView#4]
I'm trying applying light font like this:
```
let regular = UIFont.systemFont(ofSize: 15)
let light = UIFont.systemFont(ofSize: 15, weight: UIFontWeightLight)
label.attributedText = "regular ".font(regular)
                                         .append("light").font(light)
                                         .attributedText
```
But both words are with the same font (regular).

**Comment by ziryanov** [comment]
I've created pull request with fix. Please check and merge

**Comment by evermeer** [comment]
Thanks! Your PR has been merged and published as version 0.5.1
